src/sentry/static/sentry/app/views/organizationEvents/index.jsx

```diff
@@ -18,13 +16,16 @@ import TimeRangeSelector from 'app/components/organizations/timeRangeSelector';
 import space from 'app/styles/space';
 import withOrganization from 'app/utils/withOrganization';
 
+import {getParams} from './utils/getParams';
```

Comment:
Was this an accident? Path hasn't changed

Reply:
yeah thanks 4bf1eedcb93b862